src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import {MovieCard} from "../MovieCard";
+import {Movie} from "../../types/movie";
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+}
+
+export const MoviesList: React.FC<Props> = ({movies}) => (
+  <ul className='movies'>
+    {movies.map(movie => (
+      <MovieCard movie={movie}/>
```

Comment:
на кожен елемент при використанні методу **мар** ми повинні  прописувати унікальний ключ
```suggestion
      <MovieCard movie={...movie} key={movie.imdbId}/>
```